app/src/main/java/fr/free/nrw/commons/review/ReviewActivity.java

```diff
@@ -217,6 +218,12 @@ private void updateImage(Media media) {
             return;
         }
 
+        //If The Media User and Current Session Username is same then Skip the Image
+        if (media.getUser() != null && media.getUser().equals(AccountUtil.getUserName(getApplicationContext()))) {
+            runRandomizer();
+            return;
+        }
```

Comment:
I guess this is part of another PR which has been merged 3 days ago.
Please rebase or pull from main. Thanks! :-)